package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -8,6 +8,7 @@ PROG=/usr/sbin/dnsmasq
 
 ADD_LOCAL_DOMAIN=1
 ADD_LOCAL_HOSTNAME=1
+ADD_LOCAL_GATENAME=0
```

Comment:
What is the meaning of `GATENAME`? Short hand for gateway hostname? If yes, why is it added for every non-dhcp interface?

Do we even need the distinction? You could make `ADD_LOCAL_HOSTNAME` control adding host names for both DHCP and no-DHCP interfaces.